docs/api-status.json

```diff
@@ -312,6 +312,14 @@
         "name": "UserPerm.Destroy",
         "comment": "Destroy will explicitly free ceph resources associated with the UserPerm.\n\nImplements:\n void ceph_userperm_destroy(UserPerm *perm);\n"
       }
+    ],
+    "preview_api": [
+      {
+        "name": "MountInfo.SelectFilesystem",
+        "comment": "SelectFilesystem to be mounted. If the ceph cluster supports more than one\ncephfs this optional function selects which one to use. Can only be called\nprior to calling Mount. The name of the file system is not validated by this\ncall - if the supplied file system name is not valid then only the\nsubsequent mount call will fail.\n\nImplements:\n int ceph_select_filesystem(struct ceph_mount_info *cmount, const char *fs_name);\n",
```

Comment:
Aren't we missing spaces in the first sentence of comment?

```
"comment": "Select file system to be mounted.
```

Reply:
Go documentation style requires the first word of a "doc comment" to be the name of the function or type you are documenting. Therefore "SelectFilesystem" since that's the name of the function. It also encourages you to make clear english sentences. THere are a bunch of examples in the go stdlib. See also https://go.dev/doc/comment#func
I could change it to "SelectFilesystem selects the file system to be mounted. ..." I just figured most people would mentally insert the space.  Let me know if you feel this still needs to be changed.

Reply:
Oh..all this while I failed to notice such a requirement. But I would still hang on to the point that encourages to make clear(and complete) English sentences(if you could).

Reply:
OK, I wasn't so wedded to the previous wording so I updated it.

Reply:
Thanks.